### ISSUE TITLE
FIREBREAK: Look into Anthos to solve multi cluster and long term multi cloud solution 

### ISSUE THREAD

**Issue by sablumiah** [ministryofjustice/cloud-platform#1962]
https://cloud.google.com/blog/topics/anthos/multi-cloud-features-make-anthos-on-aws-possible

**Comment by kcbotsh** [comment]
relies on having GPC account.

**Comment by AntonyBishop** [comment]
GCP account still unresolved.

**Comment by AntonyBishop** [comment]
GCP account now available. Unblocked.

**Comment by davidread** [comment]
At our workshop on [Resiliency against AWS service issues](https://docs.google.com/document/d/13iN6ZOoK7R89NtAsccKoum7tOMAbMn3LwmQePn74Vqk/edit#) we agreed that this is an avenue we should explore

**Comment by jasonBirchall** [comment]
Initial questions asked:
How easy is it to...
- Spin up an Anthos cluster in AWS
- Add a cloud-platform EKS cluster to an anthos cluster
- Manage centrally

**Comment by jasonBirchall** [comment]
Using the samples site, I was able to create a cluster in AWS
https://github.com/GoogleCloudPlatform/anthos-samples/tree/main/anthos-multi-cloud/AWS

**Comment by jasonBirchall** [comment]
General overview of Anthos
https://cloud.google.com/anthos/clusters/docs/multi-cloud

**Comment by jasonBirchall** [comment]
A key thing I'd like to note here is that you're deploying a GKE cluster inside of AWS - there is no EKS component (which isn't a bad thing). This may have been obvious to some/most, but I feel it's worth mentioning.

**Comment by jasonBirchall** [comment]
Starting to look at Anthos Config Management on my new GKE (inside of AWS) cluster.
https://cloud.google.com/anthos-config-management/docs/overview

**Comment by jasonBirchall** [comment]
Adding my GKE cluster (inside AWS) to anthos config manager 
https://cloud.google.com/anthos-config-management/docs/tutorials/create-configure-cluster

**Comment by jasonBirchall** [comment]
An exciting stretch goal of this ticket:
https://cloud.google.com/anthos-config-management/docs/how-to/clusterselectors

Have anthos configure both a native and non-native k8s cluster synchronously.

**Comment by jasonBirchall** [comment]
For my next trick ;) I'll be adding a "Cloud Platform" ready EKS cluster into my anthos "fleet"
https://cloud.google.com/anthos/clusters/docs/attached/how-to/attach-kubernetes-clusters

**Comment by jasonBirchall** [comment]
A look into how config sync works on Anthos
https://cloud.google.com/anthos-config-management/docs/config-sync-overview

**Comment by jasonBirchall** [comment]
Demo complete. Anthos is an imposing way to manage multi-cluster environments. It is my recommendation post-firebreak that we continue to pursue looking at GCP and Anthos as our multi-cloud env expands.
